python/ray/scripts/scripts.py

```diff
@@ -1094,6 +1095,21 @@ def memory(address):
     print(ray.internal.internal_api.memory_summary())
 
 
+@cli.command()
+@click.option(
+    "--address",
+    required=False,
+    type=str,
+    help="Override the address to connect to.")
+def status(address):
+    """Print cluster status, including autoscaling info."""
+    if not address:
+        address = services.find_redis_address_or_die()
+    logger.info("Connecting to Ray instance at {}.".format(address))
+    ray.init(address=address)
+    print(debug_status())
```

Comment:
Hey, wouldn't this be more useful if it were used like:

`ray status [cluster_yaml]`, similar to `ray dashboard cluster.yaml`?

Reply:
There are a number of ray commands that are intended to run on the currently active cluster such as ray memory, etc. This is consistent with those.